src/logic/EligibilityTimelineCalculator.js

```diff
@@ -67,6 +78,9 @@ export default class EligibilityTimelineCalculator {
                         );
                     }
                     break;
+                case CrimeClassification.FELONY_CLASS_A:
+                        console.log("Yes");
```

Comment:
Is this console log necessary? I remember we added logs to help with debugging. I'd remove it, now that we fixed the problem.